scipy/stats/tests/test_qmc.py

```diff
@@ -586,6 +597,20 @@ class TestHalton(QMCEngineTests):
                             [0.87746036, 0.21048664],
                             [0.37746036, 0.54381998]])
 
+    def test_workers(self):
+        ref_sample = self.reference(scramble=True)
+        engine = self.engine(d=2, scramble=True)
+        sample = engine.random(n=len(ref_sample), workers=8)
+
+        assert_allclose(sample, ref_sample, atol=1e-1)
```

Comment:
What is the point of this if the tolerance is so loose? Is it OK for them to be so different?

Reply:
It's a mistake from a copy/past. Both arrays should be equal.

Reply:
Done